src/helpers/mathjax.ts

```diff
@@ -0,0 +1,206 @@
+import { debounce } from 'lodash';
+import { isEmpty, memoize } from 'lodash/fp.js';
```

Comment:
I would prefer not to need the extension, but when bundled into assessments I get the error:
```
Module not found: Error: Can’t resolve ‘lodash/fp’
Did you mean 'fp.js'?
BREAKING CHANGE: The request 'lodash/fp' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
```

I'm sure there's some webpack config I can do to make this unnecessary but could not figure it out quickly.

Reply:
this may be because of lodash changing their bundling of the library for different module types, i wouldn't worry too much about it